### PR TITLE
clear code same as `Elastica\Status`

### PR DESCRIPTION
**Pull request by ReenExe**
in both `$this->_data = $this->getResponse()->getData();`

### CODE REVIEW

File: lib/Elastica/Cluster.php [hunk 1]
```diff
@@ -73,14 +73,7 @@ public function getResponse()
...
     {
-        $metaData = $this->_data['metadata']['indices'];
-
-        $indices = array();
-        foreach ($metaData as $key => $value) {
-            $indices[] = $key;
-        }
-
-        return $indices;
+        return array_keys($this->_data['metadata']['indices']);
```
Comment (ruflin) [inline lib/Elastica/Cluster.php]: Looks to me like it does the same. To we have a unit test that covers this? You can easily check in the coverage report.

Comment (ReenExe) [inline lib/Elastica/Cluster.php]: @ruflin, Yes exists test in \Elastica\Test\StatusTest::testGetIndexNames https://github.com/ruflin/Elastica/blob/master/test/lib/Elastica/Test/StatusTest.php#L37

Comment (ruflin) [inline lib/Elastica/Cluster.php]: Yep, looks good. That is what I was referring to: https://codecov.io/gh/ruflin/Elastica/src/bc7072afc02957035bd1c579e5dbd74e553950e4/lib/Elastica/Cluster.php#L74

**Comment by ReenExe** [comment]
`codecov` - it's new tools for me - thanks

**Pull request merged**